File: ispit_rs1-2022-10_31/webapi/FIT_Api_Examples/Modul2/ViewModels/StudentGetAllVM.cs
namespace FIT_Api_Examples.Modul2.ViewModels
{
    public class StudentGetAllVM
    {
        public int id { get; set; }
        public string ime { get; set; }
        public string prezime { get; set; }
        public int? opstina_rodjenja_id { get; set; }
        public string? opstina_rodjenja_opis { get; set; }
        public string broj_indeksa { get; set; }
        public string drzava_rodjenja_opis { get; set; }
        public string vrijeme_dodavanja { get; set; }
        public string? slika_korisnika_nova_base64 { get;  set; }

        public byte[]? slika_korisnika_postojeca_base64_DB { get; set; }
        public byte[]? slika_korisnika_postojeca_base64_FS { get; set; }
    }
}

File: ispit_rs1-2022-10_31/angular_app/src/app/studenti/student-getall-vm.ts
export interface StudentGetallVM {
  id: number;
  ime: string;
  prezime: string;
  opstina_rodjenja_id: number | null;
  opstina_rodjenja_opis: string;
  broj_indeksa: string;
  drzava_rodjenja_opis: string;
  vrijeme_dodavanja: string;
  slika_korisnika_nova_base64?: string | null;
  slika_korisnika_postojeca_base64_DB: string;
  slika_korisnika_postojeca_base64_FS: string;
  omiljenipredmeti:any;
}

export interface StudentGetallVMPagedList {
  dataItems: StudentGetallVM[];
  currentPage: number;
  totalPages: number;
  pageSize: number;
  totalCount: number;
  hasPrevios: boolean;
  hasNext: boolean;
}
